rocky/tools/forms/ooi_form.py

```diff
@@ -154,7 +154,18 @@ class ClearanceFilterForm(BaseRockyForm):
     )
 
 
-SORTED_OOI_TYPES = sorted([ooi_class.get_ooi_type() for ooi_class in get_collapsed_types()])
+class OOITypeExcluded(Enum):
+    FINDING = "Finding"
+    FINDING_TYPE = "FindingType"
+
+
+SORTED_OOI_TYPES = sorted(
+    [
+        ooi_class.get_ooi_type()
+        for ooi_class in get_collapsed_types()
+        if ooi_class.get_ooi_type() not in [excluded_types.value for excluded_types in OOITypeExcluded]
+    ]
+)
```

Comment:
Interesting! Although this method works, I think it can be much simpler. Since you're essentially dealing with strings, you can streamline the process by directly comparing the strings. Converting them to enums and back seems unnecessary within this context.

You could do something like:
```suggestion
_EXCLUDED_OOI_TYPES = ("Finding", "FindingType")

SORTED_OOI_TYPES = sorted(
    [
        ooi_class.get_ooi_type()
        for ooi_class in get_collapsed_types()
        if ooi_class.get_ooi_type() not in _EXCLUDED_OOI_TYPES
    ]
)
```